packages/scope-manager/tests/util/serializers/TSESTreeNode.ts

```diff
@@ -18,7 +18,7 @@ const SEEN_NODES = new Map<Node, number>();
 
 const serializer: NewPlugin = {
   test(val): boolean {
-    return (
+    return !!(
```

Comment:
sigh. if `unknown` didn't refine to `object`.. we could just annotate this easily.
The alternative here is to create a simple type guard function:
```ts
function isObject(val: unknown): v is Record<string, unknown> {
  return val && typeof val === 'object';
}
```

Reply:
Yeah... I don't mind having the `!!`s in there though, since they force it to be the `boolean` value we expect it to be anyway. Thoughts?